Source/WrapDelphiSamplesSpin.pas

```diff
@@ -0,0 +1,100 @@
+{$I Definition.Inc}
+
+unit WrapDelphiSamplesSpin;
+
+interface
+
+uses
+  Classes, Vcl.Samples.Spin, WrapDelphi, WrapDelphiControls, WrapDelphiStdCtrls;
+
+type
+  TPyDelphiSpinButton = class (TPyDelphiWinControl)
+  private
+    function  GetDelphiObject: TSpinButton;
+    procedure SetDelphiObject(const Value: TSpinButton);
+  public
+    class function  DelphiObjectClass : TClass; override;
+    // Properties
+    property DelphiObject: TSpinButton read GetDelphiObject write SetDelphiObject;
+  end;
+
+  TPyDelphiSpinEdit = class (TPyDelphiCustomEdit)
+  private
+    function  GetDelphiObject: TSpinEdit;
+    procedure SetDelphiObject(const Value: TSpinEdit);
+  public
+    class function  DelphiObjectClass : TClass; override;
+    // Properties
+    property DelphiObject: TSpinEdit read GetDelphiObject write SetDelphiObject;
+  end;
+
+implementation
+
+{ Register the wrappers, the globals and the constants }
+type
+  TSamplesSpinRegistration = class(TRegisteredUnit)
+  public
+    function Name: string; override;
+    procedure RegisterWrappers(APyDelphiWrapper: TPyDelphiWrapper); override;
+    procedure DefineVars(APyDelphiWrapper: TPyDelphiWrapper); override;
+  end;
+
```

Comment:
Why include DefineVars?

Reply:
Oh, well, not for a particularly good reason.
I followed the pattern(s) from other DelphiWrap units and at least one of those also had this empty DefineVars procedure. It can be left out but it also doesn't do any harm I guess.

Reply:
No problem.  Thanks!